conf/local.conf.append

```diff
@@ -0,0 +1,3 @@
+# Comment to disable and Uncomment to enable java feature
+USER_FEATURES += "java"
+EXTRA_IMAGE_FEATURES:append:feature-java = " java"
```

Comment:
@muhamzai Is java a valid image feature? I could not find FEATURE_PACKAGES_java in meta-java layer. See how we created virtualization image feature https://github.com/MentorEmbedded/meta-sokol-flex/blob/kirkstone/meta-sokol-flex-distro/conf/distro/sokol-flex.conf#L249

Reply:
Its created in this pull request.